README.md

```diff
@@ -2,62 +2,34 @@
 
 ## Overview
 
-This project contains the Python code and SQL schema for the Louis project
+This project contains the code for the backend of the louis project, tying the database to both louis-frontend and louis-backend
```

Comment:
tying to itself itself too? because this is louis-backend

Reply:
done